test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SdkTypeBindings/DotNetIsolated_BlobStorage.cs

```diff
@@ -0,0 +1,241 @@
+            var responseContent = await response.Content.ReadAsStringAsync();
+
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            Assert.Equal(inputText, responseContent);
+        }
+
+        [Fact]
+        public async Task HttpTrigger_BlobInputStreamFunction_Invoke_Succeeds()
+        {
+            var inputText = "BlobInputStreamFunction - hello world";
+
+            CloudBlockBlob testBlob = Fixture.TestInputContainer.GetBlockBlobReference("sample1.txt");
+            await testBlob.UploadTextAsync(inputText);
+
+            var response = await SamplesTestHelpers.InvokeHttpTrigger(Fixture, "BlobInputStreamFunction");
+            var responseContent = await response.Content.ReadAsStringAsync();
+
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            Assert.Equal(inputText, responseContent);
+        }
+
+        [Fact]
+        public async Task HttpTrigger_BlobInputByteArrayFunction_Invoke_Succeeds()
+        {
+            var inputText = "BlobInputByteArrayFunction - hello world";
+
+            CloudBlockBlob testBlob = Fixture.TestInputContainer.GetBlockBlobReference("sample1.txt");
+            await testBlob.UploadTextAsync(inputText);
+
+            var response = await SamplesTestHelpers.InvokeHttpTrigger(Fixture, "BlobInputByteArrayFunction");
+            var responseContent = await response.Content.ReadAsStringAsync();
+
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            Assert.Equal(inputText, responseContent);
+        }
+
+        [Fact]
+        public async Task HttpTrigger_BlobInputStringFunction_Invoke_Succeeds()
+        {
+            var inputText = "BlobInputStringFunction - hello world";
+
+            CloudBlockBlob testBlob = Fixture.TestInputContainer.GetBlockBlobReference("sample1.txt");
+            await testBlob.UploadTextAsync(inputText);
+
+            var response = await SamplesTestHelpers.InvokeHttpTrigger(Fixture, "BlobInputStringFunction");
+            var responseContent = await response.Content.ReadAsStringAsync();
+
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            Assert.Equal(inputText, responseContent);
+        }
+
+        [Fact]
+        public async Task HttpTrigger_BlobInputBookFunction_Invoke_Succeeds()
+        {
+            var inputBook = @"{ ""id"": ""1"", ""name"": ""Golden Compass"" }";
+
+            CloudBlockBlob testBlob = Fixture.TestInputContainer.GetBlockBlobReference("book.json");
+            await testBlob.UploadTextAsync(inputBook);
+
+            var response = await SamplesTestHelpers.InvokeHttpTrigger(Fixture, "BlobInputBookFunction");
+            var responseContent = await response.Content.ReadAsStringAsync();
+
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            Assert.Equal("Golden Compass", responseContent);
+        }
+
+        [Fact]
+        public async Task BlobTrigger_BlobClientFunction_Invoke_Succeeds()
```

Comment:
All of the BlobTrigger tests pass when run individually but are flaky when the entire class runs all the tests together - the issue being that no logs are found. Looking for some suggestions to either make these 5 tests run one after the other or an alternative way to validate these BlobTrigger functions

Reply:
Are they failing when just running the entire class locally or from inside the integration test pipeline?

Reply:
Are they failing when just running locally or from inside the integration tests pipeline?

Reply:
Locally, but could also be an issue in the pipelines - trying to get the pipeline to work to check there too. Either way I want to make sure I don't merge any flaky tests

Reply:
Looks like xunit supports something like below for sequential execution. Source - [SO Link](https://stackoverflow.com/questions/1408175/execute-unit-tests-serially-rather-than-in-parallel), [Link](https://medium.com/selectstarfromweb/xunit-tests-in-parallel-ad32788ce1bd). There are more suggestions in SO link.

```csharp
namespace IntegrationTests
{
    [Collection("Sequential")]
    public class Class1
    {
        [Fact]
        public void Test1() { }

        [Fact]
        public void Test2() { }
    }
}

```

Reply:
My understanding from that SO discussion is that all of the individual tests in the the class will still run in parallel, but the classes will run in sequence - meaning I would need to create a class for each test for them to run in sequence?